plugins/modules/zabbix_maintenance.py

```diff
@@ -301,14 +311,24 @@ def main():
     else:
         zabbix_host = "host"
 
+    if not name:
+        name = f"Created at {now.strftime('%Y-%m-%d %H:%M')}"
```

Comment:
unfortunately, f-strings are supported starting form 3.6+, we need to also still support python2.7 until ansible 2.12 is deprecated:
```
 ERROR: Found 1 compile issue(s) on python 2.7 which need to be resolved:
ERROR: plugins/modules/zabbix_maintenance.py:315:61: SyntaxError: name = f"Created at {now.strftime('%Y-%m-%d %H:%M')}"
```